.github/workflows/ci.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
         id: go
         uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
         with:
-          go-version: '1.20'
+          go-version: '1.20.4'
```

Comment:
you dont need to do this for the CI jobs, the action get the latest go available already

Reply:
same for all the other places you changed in the ci jobs

Reply:
Thanks for the suggestion.